clomonitor-core/src/linter/checks/trademark_disclaimer.rs

```diff
@@ -19,7 +19,7 @@ pub(crate) const CHECK_SETS: [CheckSet; 1] = [CheckSet::Community];
 lazy_static! {
     #[rustfmt::skip]
     pub(crate) static ref TRADEMARK_DISCLAIMER: RegexSet = RegexSet::new(vec![
-        r"https://(?:w{3}\.)?linuxfoundation.org/trademark-usage",
+        r"https://(?:w{3}\.)?linuxfoundation.org/(?:legal/)?trademark-usage",
```

Comment:
Could you please add a new entry to the test function below with the new url?

Reply:
Thanks for the catch, @tegioz  🙇  I think I've got the test assertions updated appropriately, please advise if you'd prefer a different strategy.